### ISSUE TITLE
Unexpected space error

### ISSUE THREAD

**Issue by Mottie** [douglascrockford/JSLint#6]
I use JSLint to check my jQuery scripts and if I check code like this using "The Good Parts", it only tells me jQuery isn't defined - which is fine:

```
jQuery('.selector')
  .removeAttr('style')
  .show();
```

Now if I define an action and use it this way:

```
var showOrHide = 'show';
jQuery('.selector')
  .removeAttr('style')
  [showOrHide]();
```

I get an additional error stating `Unexpected space before '['.`

The only way to get rid of this error is to format the script like this, which I think make it more unreadable:

```
var showOrHide = 'show';
jQuery('.selector')
  .removeAttr('style')[showOrHide]();
```

So my question is would it be possible to make the second example not produce an error?

Thanks :)
